packages/server/lib/helpers/email.ts

```diff
@@ -67,3 +67,24 @@ Team Nango</p>
             `
     );
 }
+
+export async function sendTrialAlmostOverEmail({ user, inDays }: { user: Pick<DBUser, 'name' | 'email'>; inDays: number }) {
+    const emailClient = EmailClient.getInstance();
+    await emailClient.send(
+        user.email,
+        `Your Nango trial ends in ${inDays} days`,
+        `<p>Your trial ends in ${inDays} days, ${user.name}</p>
+
+<p>We hope you were able to spend the last two weeks exploring how Nango can help you save time and increase your productivity!</p>
```

Comment:
I would choose a phrasing with no mention of two weeks so when we change the trial duration we don't have to change the email 😁